apps/web/src/components/Settings/Export/Publications.tsx

```diff
@@ -81,7 +81,7 @@ const Publications: FC = () => {
           Export all your posts, comments and mirrors to a JSON file.
         </Trans>
       </div>
-      {publications.length > 0 ? (
+      {publications && publications.length > 0 ? (
```

Comment:
use optional chaining here instead of &&

Reply:
pushed